infrastructure/ci/development/deployment-config.template.yml

```diff
@@ -82,6 +82,26 @@ Resources:
                   - ssm:GetParameter
                   - ssm:GetParameters
                   - ssm:GetParametersByPath
+              
+              - Effect: Allow
+                Resource: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
```

Comment:
Region and accountId should not be hardcoded, example:
`Resource: !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${ServiceName}/*`

Reply:
Can you provide reasoning for this?
Otherwise I see no reason to parameterise this.